### PR TITLE
Update docs to use LocatorAwareTrait::getTable().

### PR DESCRIPTION
**Pull request by ADmad**
Refs cakephp/cakephp#15947

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @markstory

File: en/console-commands/commands.rst [hunk 1]
```diff
@@ -323,8 +323,7 @@ conventions. Let's continue by adding more logic to our command::
         public function execute(Arguments $args, ConsoleIo $io)
         {
             $table = $args->getArgument('table');
-            $this->loadModel($table);
-            $this->{$table}->query()
+            $this->getTable({$table})->query()
```
Comment (markstory) [inline en/console-commands/commands.rst]:
```suggestion
            $this->getTable($table)->query()
```

File: en/console-commands/commands.rst [hunk 2]
```diff
@@ -420,12 +419,11 @@ Update the command class to the following::
         public function execute(Arguments $args, ConsoleIo $io)
         {
             $table = $args->getArgument('table');
-            $this->loadModel($table);
             if ($io->ask('Are you sure?', 'n', ['y', 'n']) === 'n') {
                 $io->error('You need to be sure.');
                 $this->abort();
             }
-            $this->{$table}->query()
+            $this->getTable({$table})->query()
```
Comment (markstory) [inline en/console-commands/commands.rst]:
```suggestion
            $this->getTable($table)->query()
```

File: en/views/cells.rst [hunk 3]
```diff
@@ -219,14 +218,12 @@ messages could look like::
...
         {
-            $this->loadModel('Messages');
-
             // Create a paginator
             $paginator = new Paginator();
 
             // Paginate the model
             $results = $paginator->paginate(
-                $this->Messages,
+                $this->getTable('Messages'_,
```
Comment (markstory) [inline en/views/cells.rst]:
```suggestion
                $this->getTable('Messages'),
```

**Comment by ADmad** [comment]
Will reopen after 4.next merge.

**Pull request closed without merging**